cluster/mesos/docker/km/Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN apt-get update -qq && \
     apt-get clean
 
 COPY ./bin/* /usr/local/bin/
-ADD ./opt/mesos-cloud.conf /opt/
+COPY ./opt/* /opt/
```

Comment:
I think I was being explicit so the dockerfile would fail if we didn't have all the expected files, but it's not a big deal. COPY is the right command to use here at least.